ddtrace/llmobs/_llmobs.py

```diff
@@ -717,6 +721,47 @@ def submit_evaluation(
             }
         )
 
+    @classmethod
+    def inject_distributed_headers(cls, request_headers: Dict[str, str], span: Optional[Span] = None) -> Dict[str, str]:
+        """Injects the span's distributed context into the given request headers."""
+        if cls.enabled is False:
+            log.warning(
+                "LLMObs.inject_distributed_headers() called when LLMObs is not enabled. "
+                "Distributed context will not be injected."
+            )
+            return request_headers
+        if not isinstance(request_headers, dict):
+            log.warning("request_headers must be a dictionary of string key-value pairs.")
+            return request_headers
+        if span is None:
+            span = cls._instance.tracer.current_span()
+        if span is None:
+            log.warning("No span provided and no currently active span found.")
```

Comment:
Correct, we do not care if the span is LLM type, because a user might be using manual APM spans as well. It just matters that we inject the current active span's context (including LLMobs parent ID) into the request headers.